browser/src/UI/components/ToolTip.tsx

```diff
@@ -19,16 +19,16 @@ export interface IToolTipsViewProps {
 export class ToolTipsView extends React.PureComponent<IToolTipsViewProps, {}> {
 
     public render(): JSX.Element {
-        const toolTipElements = this.props.toolTips.map((toolTip) => {
-            return <ToolTipView {...toolTip} borderColor={this.props.borderColor} foregroundColor={this.props.foregroundColor} backgroundColor={this.props.backgroundColor}/>
+        const toolTipElements = this.props.toolTips.map((toolTip, i) => {
+            return <ToolTipView {...toolTip} key={`${i}-${toolTip.id}`} borderColor={this.props.borderColor} foregroundColor={this.props.foregroundColor} backgroundColor={this.props.backgroundColor}/>
```

Comment:
For this key - could we just use `toolTip.id` without the `i` parameter? Some of the UI in the tool tips is sensitive to repaints (like having both the _rename_ and _quickinfo_ dialog open). There's actually a bug now where if you open Rename, and the QuickInfo pops up, the Rename gets re-rendered and the focus gets messed up... If we just use `toolTip.id`, I believe it will fix that bug.